dist/vanilla-back-to-top.umd.js

```diff
@@ -16,7 +16,7 @@ function _typeof(obj) { if (typeof Symbol === "function" && typeof Symbol.iterat
   function addBackToTop() {
     var params = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : {};
     var _params$id = params.id,
-        id = _params$id === void 0 ? 'back-to-top' : _params$id,
+        id = _params$id === void 0 ? 'back-to-top-' + Date.now() : _params$id,
```

Comment:
this will break the 4th example here: https://github.com/vfeskov/vanilla-back-to-top/blob/master/EXAMPLES.md

how would such id be useful? a new unique one is generated every page load, so it can't be targeted via js or css, can it?